### ISSUE TITLE
MAS 3.1.1: <html> element must have a lang attribute (#iframe;html)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4208]
Azure Notebooks - Welcome has an accessibility issue that needs investigation.

Issue Details
<html> element must have a lang attribute
html-has-lang

Element Path
#iframe;html

Snippet
<html>

How to fix
Fix any of the following:
- The <html> element does not have a lang attribute

To reproduce
Use K4W to investigate the issue details

Environment
Chrome version 69.0.3497.100

This accessibility issue was found using Keros for Web, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/K4Web.

[MAS 3.1.1 html element must have a lang attribute (#iframe;html).zip](https://github.com/jupyter/notebook/files/2576222/MAS.3.1.1.html.element.must.have.a.lang.attribute.iframe.html.zip)

**Comment by joshuazeltser** [comment]
Using the K4Web tool above on both Mac and Windows the tool finds a valid lang attribute as shown here:
![image](https://user-images.githubusercontent.com/17781896/60174558-343c3780-9809-11e9-89d2-6f2607021fc5.png)

Based on this I don't see the issue here?

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
